extensions/ql-vscode/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [UNRELEASED]
 
+- Add CodeLens feature to make the CodeQL quick evaluation command more accessible. [#1035](https://github.com/github/vscode-codeql/pull/1035)
```

Comment:
I've attempted to word this in a way that tells the user how to use the feature. What do you think?
```suggestion
- Add a CodeLens to make the Quick Evaluation command more accessible. Click the `Quick Evaluation` prompt above a predicate definition in the editor to evaluate that predicate on its own. [#1035](https://github.com/github/vscode-codeql/pull/1035)
```

Reply:
I like it. I tried to make it concise, but I should have included more information.